server/package.json

```diff
@@ -35,11 +35,13 @@
     "express": "^5.0.0-alpha.8",
     "express-session": "^1.17.2",
     "graphql": "^15.5.0",
+    "js-base64": "^3.6.1",
```

Comment:
Could the encoding and decoding of a b64 string you're using this library for be done using Buffer instead?

Reply:
Yes, definitely. The con is that it adds a dependency. There are two pros I think: the biggest plus for me is the fact that the characters it uses are the letters and numbers only (it doesn't insert `/` or `=` characters like Node does), which works better in IDs, and the second biggest plus is that the API is more readable.